test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -76,6 +76,12 @@ public void All_props_and_targets_add_themselves_to_MSBuildAllTargets()
                 return;
             }
 
+            //  Disable this test when using full Framework MSBuild, as the paths to the props and targets are different
+            if (UsingFullFrameworkMSBuild)
```

Comment:
We're currently doing it like this to skip tests on non-Windows, and perhaps in other cases.  I've filed #736 for actually marking these as skipped.